ext/standard/config.w32

```diff
@@ -1,6 +1,18 @@
 // vim:ft=javascript
 // $Id$
 
+ARG_WITH("argon2", "Argon2 support", "no");
+
+if (PHP_ARGON2 != "no") {
+	if (CHECK_LIB("Argon2Ref.lib", "argon2", PHP_ARGON2)
+	&& CHECK_HEADER_ADD_INCLUDE("argon2.h", "CFLAGS_ARGON2")) {
+		AC_DEFINE('HAVE_ARGON2LIB', 1);
+		EXTENSION("argon2", null, false);
```

Comment:
Hi! It's been a while since I was tinkering around with this, but from vague memory this is how it works:

CHECK_LIB':
 - The second parameter is the target of where to link the Argon2Ref.lib to (as per extension), this can be set to empty ("") -- See win32\build\confutils.js
 - The third parameter is fine (as it uses PHP_ARGON2, which is the one from the --with-argon2=XXX)

CHECK_HEADER_ADD_INCLUDE
 - This sets the compiler flags for an extension (CFLAGS_ARGON2), and should simply be CFLAGS (we use this value for core things, like determining IPv6 support in win32\build\config.js).

TL;DR:
Line 7 should be: if (CHECK_LIB("Argon2Ref.lib", "", PHP_ARGON2)
Line 8 should be: && CHECK_HEADER_ADD_INCLUDE("argon2.h", "CFLAGS")) {

which should allow you to drop the EXTENSION() call.

(bear in mind I'm rusty, but else poke around the sources in win32\build\)!